### PR TITLE
Skip apt repo publish if not stable tag

### PR DESCRIPTION
**Pull request by develohpanda**
Closes INS-742

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -8,19 +8,36 @@ on:
...
+
   apt_repository:
     name: Publish to apt repository
     runs-on: ubuntu-latest
+    needs: extract_channel
+    if: needs.extract_channel.outputs.channel == 'stable'
 
     steps:
     - name: Download .deb from GitHub release
+      if: steps.stable-tag.outputs.is-stable == 'true'
```
Comment (DMarby) [inline .github/workflows/publish.yml]: I guess this is not needed anymore since the check is moved to a job instead? Same for the one below

**Review by DMarby — CHANGES_REQUESTED** [review]

**Review by DMarby — APPROVED** [review]

**Comment by develohpanda** [comment]
@DMarby I tested this and it didn't exit/continue properly 😆 will need to pair briefly

**Comment by DMarby** [comment]
> @DMarby I tested this and it didn't exit/continue properly 😆 will need to pair briefly

Forgot that we were doing POSIX regex when reviewing 😅 `\d` isn't supported, so we'll need to change it to this instead and it should work:
```
^core@[0-9]{4}\.[0-9]+\.[0-9]+$
```

**Comment by develohpanda** [comment]
Looks like it's working now!

**Pull request merged**